careers/ladder.md

```diff
@@ -64,178 +63,212 @@ Scope of impact:   Self   >   Team   > Teams > Org
-  projects.
-- Effectively contribute to longer-term product strategy.
-- Empowers team members to take on meaty and interesting tasks.
-- Can align a small group of people outside of their immediate team to solve a well defined problem.
-- Mentors junior engineers and / or train other engineers.
-- Participates in decision making at the practice level.
-
-#### Discretion
-
-- Requires minimal direction or oversight, delivers quickly.
-- Understands elements of the business that their work supports.
-- Demonstrates fluency with team KPIs and key drivers of these KPIs.
-- Makes tradeoffs between business impact and technical strategy.
-
-### IC6 Staff Engineer
-
-#### tl;dr: can drive change and impact across multiple teams
-
-#### Knowledge Leadership
-
-- Stewards large scale and impactful service or collection of services.
-- Demonstrates advanced knowledge in a specific domain and some minimum breadth across the whole stack.
-
-#### Impact
-
-- Anticipates technical issues at the product level and makes architectural and design decisions to avoid them.
-- Has track record of creating sweeping improvements in stability, performance, and scalability across services.
-- Consistently reduces the complexity of projects, services, and processes in order to get more done with less
-  work.
-- Has made an obvious positive impact on the entire company's technical trajectory.
-
-#### Influence
-
-- Sought out by members of other teams for technical guidance.
-- Recognized as a prolific contributor.
-- Multiplies the effectiveness of others by facilitating cross-team work.
-- Can communicate with empathy at scale.
-- Can drive change at the org level by identifying a critical need, spinning up a practice / working group / task
-  force and seeing to completion.
-
-#### Discretion
-
-- Understands the Artsy business and makes decisions based on its direction.
-- Sets short to medium-term strategic technical direction and positively impacts business direction.
-
-### IC7 Senior Staff Engineer
-
-#### tl;dr: can drive change and impact across full Product, Data, Design and Engineering organization
-
-#### Knowledge Leadership
-
-- Creates architecture that enables many potential futures without knowing exactly what the future is.
-- Quickly breaks down complex problems into potential solutions, knowns, and unknowns.
-- Leads the conversations internally about the direction of major areas of the technology.
-
-#### Impact
-
-- Consistently delivers large systems involving one or more teams' contribution.
-
-#### Influence
-
-- Primarily acts as a multiplier by building systems, authoring tools, or introducing policies or patterns.
-
-#### Discretion
-
-- Ensures Artsy makes judicious technology choices in the face of rapid business and technology advancements.
-- Identifies technology strategic growth opportunities that enable Artsy to grow as a business.
+### Capabilities
+
+<style>
```

Comment:
@sweir27 actually Github doesn't seem to process your style tag properly.

Reply:
Good call, fixed in a separate PR.